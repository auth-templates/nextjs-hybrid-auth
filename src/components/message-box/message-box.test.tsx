import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import renderer from 'react-test-renderer';
import MessageBox from './message-box';

describe('MessageBox', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<MessageBox />).toJSON();
        expect(tree).toMatchSnapshot();
    });
})