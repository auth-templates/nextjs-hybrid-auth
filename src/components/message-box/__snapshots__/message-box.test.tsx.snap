// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageBox renders correctly 1`] = `
<ul
  aria-live="assertive"
  className="info"
  role="alert"
/>
`;
